.storybook/main.js

```diff
@@ -1,5 +1,16 @@
 module.exports = {
-  stories: ['../src/**/*.stories.[tj]s', '../src/**/*.stories.[tj]sx'],
+  stories: [
+    {
+      directory: '../src/ui',
+      titlePrefix: 'Design System/Components/',
```

Comment:
@karelianpie I wanted these titles to be inferred rather than need to be entered by hand:

The benefits:
- less code to write,
- enforce consistent naming at a directory level rather than by this attribute.
- make our stories simpler

Reply:
I definitely think this is a great improvement 👍